docs/en/index.rst

```diff
@@ -4,16 +4,40 @@
 Italian eIDAS Wallet Technical Specifications
 =============================================
 
-[TODO INTRO]
+Introduction
+------------
 
-Introduzione
+**What is eIDAS**
 
-cos'è eIDAS
+eIDAS stands for "electronic IDentification, Authentication and trust Services" and is the commonly used name for the EU Regulation, 910/2014, on electronic identification and trust services for electronic transactions in the internal market, repealing the old signature Directive 1999/93/EC.
 
-cos’è IT-Wallet
+The new eIDAS project establishes the European Digital Identity Wallet (EUDI Wallet) and proposes to overcome the dissimilarities, both in technological and user experience terms, that exist among the 21 digital identities (eIDs) currently active within as many as 16 Member States.
 
-scopo delle regole tecniche
+**What is IT Wallet**
 
+The IT Wallet project was created to improve the national digital identity governance experience and respond to the input received from the European community in the areas of innovation, decentralization and digital awareness. 
+
+To date, three identity systems coexist in Italy for access to public and private web services, namely: 
```

Comment:
this is not of interest about what EUDI IT wallet is

@ruphy WDYT?

I would not take this in the introduction

Reply:
I don't see why not -- it can provide meaningful context and it's quite succint. I would keep it!